doc/start/quick-rbd.rst

```diff
@@ -66,4 +66,4 @@ See `block devices`_ for additional details.
 .. _block devices: ../../rbd
 .. _FAQ: http://wiki.ceph.com/How_Can_I_Give_Ceph_a_Try
 .. _OS Recommendations: ../os-recommendations
-.. _rbdmap manpage: ../../man/8/rbdmap
+.. _rbdmap-generator manpage: ../../man/8/rbdmap-generator
```

Comment:
use `:program:` if you please. the relative path is fragile.

Reply:
I changed this to ".. _rbdmap-generator manpage: :program:\`rbdmap-generator\`", but the link is broken. Can you suggest how exactly to use `:program:` in the link?

Reply:
I've read some docs about sphinx, it seems `:program:` cannot be used to generate links. Now I use `:doc:` with absolute path. At least we can get a warning if the link breaks in the future.

Reply:
sorry, i was wrong. `:program:` is not supposed to be used here. yeah, `:doc:` is more appropriate.